projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -215,7 +218,14 @@ export class IgxDropDownBase implements OnInit, IToggleView {
      */
     public get selectedItem(): any {
         const selection = this.selectionAPI.get_selection(this.id);
-        return selection && selection.length > 0 ? selection[0] as IgxDropDownItemComponent : null;
+        const selectedItem = selection && selection.length > 0 ? selection[0] as IgxDropDownItemComponent : null;
```

Comment:
If you initialize the selection along with the component you won't need to check if it's there.
At that point this can be just `const selectedItem = selection[0] as IgxDropDownItemComponent` as [0] of empty array will just return undefined and you already have a check for the selected item right after :)

Reply:
Yes, it will be better in this way. Fixed.